src/Pages/Main/Components/StoreDetail/Modal.js

```diff
@@ -1,42 +1,17 @@
 import styled from 'styled-components';
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { BsStarFill, BsStar, BsStarHalf } from 'react-icons/bs';
 
 const Modal = props => {
   const [starFilled, setStar] = useState([false, false, false, false, false]);
-  const [postData, setData] = useState({
-    rating: '',
-    content: '',
-    image_url: '',
-  });
+  const [postData, setPost] = useState({ store_id: props.id });
 
   const handleStarFill = index => {
     let click = [...starFilled];
     for (let i = 0; i < 5; i++) {
       i <= index ? (click[i] = true) : (click[i] = false);
     }
```

Comment:
해당 PR에서 수정된 코드는 아니지만 리뷰는 남기겠습니다!
```suggestion
    for (let i = 0; i < 5; i++) {
      click[i] = i <= index;
    }
```

혹은
```suggestion
    click.map( (star, i) => i <= index )
```

Reply:
Wow

Reply:
맵은 무슨 의미인거죠....???? 😂🤦‍♀️💭

Reply:
해결했습니다. (i <= index)가 true or false값이니까 index값을 인식해서 바뀐 배열을 반환해준다..!!! 꿀팁 감사합니다 ❤